web/forms.py

```
@@ -1869,3 +1872,58 @@ class StudyGroupForm(forms.ModelForm):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Order subjects by name
+        self.fields["category"].queryset = Subject.objects.all().order_by("name")
+
+    def clean_title(self):
+        title = self.cleaned_data.get("title", "")
+        # strip any tags entirely (just keep plain text)
+        return bleach.clean(title, tags=[], attributes={}, strip=True)
+
+    def clean_description(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean_description`

(ANN201)